app/frontend/src/submissions/components/SubmissionForm/ActivityType.vue

```diff
@@ -1,16 +1,15 @@
 <template>
   <fieldset>
-    <legend>Type of Work</legend>
+    <legend :id="id">Type of Work</legend>
```

Comment:
Probably a non-issue but I think the actual `id` text could have been put here instead of passed through from the parent ?